install.bash

```diff
@@ -0,0 +1,447 @@
+		echo -e "${GREEN}Batctl is already installed${RESET}"
+	else
+		# Install batctl
+
+		sudo apt install -y batctl
+	fi
+
+	# Check if batman-adv is installed (modinfo batman-adv)
+
+	if [[ -x "$(command -v modinfo)" ]]; then
+		if modinfo batman-adv &>/dev/null; then
+			echo -e "${GREEN}Batman-adv is already installed${RESET}"
+			MESH_SUPPORT=true
+		else
+			# Install batman-adv
+			echo -e "${YELLOW}The batman-adv kernel module is not installed. Very often it can be installed by installing the extra linux kernel packages.${RESET}"
+			read -p "Do you want to install the batman-adv kernel module? (y/n) via linux-modules-extra-$(uname -r) " -n 1 -r
+			if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+				sudo apt install linux-modules-extra-$(uname -r)
+
+				if modinfo batman-adv &>/dev/null; then
+					echo -e "${GREEN}Batman-adv installed successfully${RESET}"
+					MESH_SUPPORT=true
+
+				else
+					echo -e "${RED}Failed to install batman-adv${RESET}"
+					echo -e "${RED}Proceeding without mesh support${RESET}"
+				fi
+			fi
+		fi
+
+		# If mesh support is enabled (MESH_SUPPORT=true), activate the batman-adv kernel module
+
+		if [[ "$MESH_SUPPORT" == true ]]; then
+			# Check if batman-adv is loaded (lsmod | grep batman-adv)
+
+			if lsmod | grep batman-adv &>/dev/null; then
+				echo -e "${GREEN}Batman-adv is already loaded${RESET}"
+			else
+				# Load batman-adv
+
+				sudo modprobe batman-adv
+
+				# Also load batman-adv at boot
+
+				echo -e "${GREEN}Enabling batman-adv to run at boot${RESET}"
+				sudo sh -c "echo batman-adv >> /etc/modules"
+			fi
+		fi
+
+	fi
+else
+	echo -e "${RED}Skipping mesh support${RESET}"
+fi
+
+# Checking if hyved is installed
+
+if [[ -x "$(command -v hyved)" ]]; then
+	echo -e "${GREEN}Hyved is already installed${RESET}"
+else
+	echo -e "${RED}Hyved is not installed${RESET}"
+	echo -e "${YELLOW}Installing it now ${RESET}"
+	echo "${GREEN}Adding our GPG key${RESET}"
+	echo "deb https://apt.p2p.industries /" | sudo tee /etc/apt/sources.list.d/p2p-industries.list
+	wget -qO - https://apt.p2p.industries/key.gpg | sudo gpg --dearmor --yes -o /etc/apt/trusted.gpg.d/p2p-industries.gpg
+	sudo apt update
+	sudo apt install hyved
+fi
+
+echo -e "${RED}WARNING: If you choose yes in the following step, the wifi interface over which you might be connected will change its config and you won't be able to connect to it conventionally anymore. Please make sure you are connected via an alternative interface or ok with losing access. You can still access the computer with an attached monitor and keyboard or via other devices in the network.${RESET}"
+read -p "Do you want to configure the interfaces now? (y/n) " -n 1 -r
+
+echo
+
+if [[ "$REPLY" =~ ^[Yy]$ ]]; then
+	echo -e "${GREEN}Configuring interfaces${RESET}"
+	output=$(hyvectl init --config)
+	wifi_interface=$(echo $output | jq -r '.wifi_interface')
+
+	echo "Configuring interface $wifi_interface"
+
+	sudo systemctl enable --now wpa_supplicant@$wifi_interface
```

Comment:
This won't work just like this, because the `hyved` deb package just installs the file `/etc/wpa_supplicant/wpa_supplicant-wlan0.conf`. Instead the Debian package should be changed to install this file to `/usr/lib/hyved/wpa_supplicant-generic.conf` and this installation script should then copy that file to `/etc/wpa_supplicant/wpa_supplicant-$wifi_interface.conf before running the systemctl enable command.